src/wp-includes/post.php

```diff
@@ -7830,6 +7831,37 @@ function wp_queue_posts_for_term_meta_lazyload( $posts ) {
 			$post_type_taxonomies[ $post->post_type ] = get_object_taxonomies( $post->post_type );
 		}
 
+		foreach ( $post_type_taxonomies[ $post->post_type ] as $taxonomy ) {
+			$prime_post_terms[ $taxonomy ][] = $post->ID;
+		}
+	}
+
+	if ( $prime_post_terms ) {
+		$prime_term_ids = array();
+		foreach ( $prime_post_terms as $taxonomy => $post_ids ) {
+			$_term_ids = wp_cache_get_multiple( $post_ids, "{$taxonomy}_relationships" );
+			if ( is_array( $_term_ids ) ) {
+				$_term_ids = array_filter( $_term_ids );
+				foreach ( $_term_ids as $term_ids ) {
+					// Backward compatibility for if a plugin is putting objects into the cache, rather than IDs.
+					foreach ( $term_ids as $term_id ) {
+						if ( is_numeric( $term_id ) ) {
+							$prime_term_ids[] = (int) $term_id;
+						} elseif ( isset( $term_id->term_id ) ) {
+							$prime_term_ids[] = (int) $term_id->term_id;
+						}
+					}
+				}
+			}
+		}
+
+		if ( $prime_term_ids ) {
+			$prime_term_ids = array_unique( $prime_term_ids );
+			_prime_term_caches( $prime_term_ids );
```

Comment:
```suggestion
			_prime_term_caches( $prime_term_ids, false );
```

Let's not prime the meta cache now.

Reply:
We can probably to that here, but for that we also have to fix `_prime_term_caches()` first as it has the same issue as `_prime_post_caches()`, highlighted in #3658.